firebase-functions/test/helper.ts

```diff
@@ -41,9 +45,10 @@ export function createNext(): NextFunction {
 }
 
 export function createAdminAuth(options: CreateAdminOptions = {}) {
-  const { claimsSpy, updateUserSpy, verifyIdTokenSpy } = options;
+  const { getUserSpy, claimsSpy, updateUserSpy, verifyIdTokenSpy } = options;
```

Comment:
These look like they were in alphabetical order before this change.

Reply:
```suggestion
  const { claimsSpy, getUserSpy, updateUserSpy, verifyIdTokenSpy } = options;
```